sql_metadb/derived_tables/po_lines_location.sql

```diff
@@ -17,25 +18,51 @@ WITH ploc AS (
 )
 SELECT
     ploc.pol_id,
-    ploc.pol_loc_id,
     ploc.pol_loc_qty,
     ploc.pol_loc_qty_elec,
     ploc.pol_loc_qty_phys,
-    loc.name AS location_name
+CASE WHEN ploc.pol_loc_id IS NOT NULL THEN ploc.pol_loc_id 
+          ELSE hr.permanent_location_id
+    END AS pol_location_id, 
+    CASE WHEN loc.name IS NOT NULL THEN loc.name
+         ELSE loc2.name
+    END AS pol_location_name,
+    CASE WHEN loc.name IS NOT NULL THEN 'pol_location'
+         WHEN loc2.name IS NOT NULL THEN 'pol_holding'
+         ELSE 'no_source'
+    END AS pol_location_source
 FROM
     ploc
-    LEFT JOIN folio_inventory.location__t AS loc ON loc.id = ploc.pol_loc_id;
+    LEFT JOIN folio_inventory.holdings_record__t AS hr ON ploc.pol_hold_id::uuid= hr.id
```

Comment:
Typo in `ploc.pol_hold_id` should be `ploc.pol_holding_id`

This prevents the query to run.